### PR TITLE
Add buttons to download and copy JSON  results

### PR DESCRIPTION
**Pull request by camillobruni**
- Add "Download JSON" button to details view
- Add "Copy JSON" button to details view
- Example JSON result file name: "speedometer_3-2023-02-13T09_18_32.318Z.json"
- Update CSS formatting to make details button all the same width
- Add "title" to results buttons (both summary and details)

### CODE REVIEW

File: resources/main.mjs [hunk 1]
```diff
@@ -269,6 +277,23 @@ class MainBenchmarkClient {
...
+    _getFormattedJSONResult()
+    {
+        const indent = "    ";
+        return JSON.stringify(this._measuredValuesList, undefined, indent);
+    }
+
+    copyJsonResults()
+    {
+
+        navigator.clipboard.writeText(this._getFormattedJSONResult());
```
Comment (rniwa) [inline resources/main.mjs:289]: This API doesn't exist in Safari or Firefox.

**Review by rniwa — CHANGES_REQUESTED** [review]
We should probably be using document.execCommand('copy') and event.clipboardData.

File: resources/main.mjs (hunk 1 above)
Comment (camillobruni) [reply resources/main.mjs:289]: It does seem to work both in Firefox 109.0.1 (64-bit) and Safari Version 16.3 on my mac.

**Comment by camillobruni** [comment]
Just look at the [mdn page again](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText#browser_compatibility), seems like the only "restriction" is that writeText is only valid during a user interaction.

Might have missed something else?

**Review by rniwa — APPROVED** [review]

**Comment by camillobruni** [comment]
Thanks 👍

**Pull request merged**